Sources/TSCBasic/Process.swift

```diff
@@ -147,6 +147,9 @@ public final class Process {
 
         /// The current OS does not support the workingDirectory API.
         case workingDirectoryNotSupported
+
+        /// The stdin could not be opened.
+        case stdinNotOpening
```

Comment:
What about `stdinUnavailable`?  "not opening" is not particularly descriptive, and makes you question what happened.

Reply:
Done.